x-pack/legacy/plugins/maps/common/i18n_getters.ts

```diff
@@ -40,6 +41,7 @@ export function getEsSpatialRelationLabel(spatialRelation) {
       return i18n.translate('xpack.maps.common.esSpatialRelation.withinLabel', {
         defaultMessage: 'within',
       });
+    // @ts-ignore
```

Comment:
Since you've converted the constants file to typescript, you might just want to give it a type there

Reply:
Actually, is the ignore even necessary without giving it a type?

Reply:
Hey @aaronjcaldwell 
The `ES_SPATIAL_RELATIONS.CONTAINS` constant were temporarly removed in https://github.com/elastic/kibana/pull/43141 .
Here is the highlighted [comment](https://github.com/elastic/kibana/pull/43141#discussion_r313537087) .
I can't also return it back since it's used as :

![image](https://user-images.githubusercontent.com/31325372/74010633-03270680-4997-11ea-89d7-c1cb65152dfa.png)


I've put there `@ts-ignore` since I'm not sure I need to remove the translation, and if the issue will be resolved, the ignore will be just removed.